Userland/Utilities/cksum.cpp

```diff
@@ -42,38 +43,48 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         paths.append("-");
 
     bool fail = false;
+    Array<u8, PAGE_SIZE> buffer;
+
     for (auto& path : paths) {
+        auto file_or_error = Core::Stream::File::open_file_or_standard_stream(path, Core::Stream::OpenMode::Read);
         auto filepath = (path == "-") ? "/dev/stdin" : path;
```

Comment:
While checking if this line was actually needed, I found out that `cksum` is doing it wrong. You can't get the actual length of the standard input by `stat`ing it and reading st_size.
If you are motivated enough, you could fix this utility.

Reply:
Nice catch! :^)